janusgraph-dist/src/assembly/static/conf/gremlin-server/gremlin-server.yaml

```diff
@@ -3,7 +3,7 @@ port: 8182
 scriptEvaluationTimeout: 30000
 channelizer: org.apache.tinkerpop.gremlin.server.channel.WebSocketChannelizer
 graphs: {
-  graph: conf/gremlin-server/janusgraph-cassandra-es-server.properties}
+  graph: conf/gremlin-server/janusgraph-berkeleyje-es-server.properties}
```

Comment:
This change will make the [Chapter 7](http://docs.janusgraph.org/0.1.0-SNAPSHOT/server.html#_using_the_pre_packaged_distribution) JanusGraph server docs incorrect, since they assume that running `bin/janusgraph.sh start` will start up Cassandra, ES, and Gremlin Server with a graph instance configured to use them.

You could make a separate server YAML for the BerkeleyJE-ES configuration that you can pass to the gremlin server CMD in the Dockerfile.